osh/hub/scan/scanner.py

```diff
@@ -487,6 +488,8 @@ def validate_options(self):
         self.client_csmock_args = self.options.get('csmock_args', None)
 
         self.metadata = self.options.get('metadata')
+        if self.metadata:
+            self.metadata = check_task_metadata(self.metadata)
```

Comment:
nit: You can move this check to `check_task_metadata` as well.

Reply:
This is reasonable in some way. I'd like to keep it as it as so the `check_task_metadata` only deals with non-null values.